### DISCUSSION TITLE
Is it possible to show a tooltip on a KryptonContextMenuItem?

### DISCUSSION THREAD

**Discussion by mbagattini** [Krypton-Suite/Standard-Toolkit#1014] [Q&A] [answered]
I'm using the KryptonRibbon control and I have several KryptonRibbonGroupButton in said ribbon. Some of these buttons have ButtonType = DropDown and are linked to a KryptonContextMenu. Finally, each KryptonContextMenu has a series of KryptonContextMenuItem.

Is there a way to show a Tooltip when the cursor rolls over one of these KryptonContextMenuItem? I have Tooltip set for main KryptonRibbonGroupButtons but I can't find a way to display additional info related to the drop down menu items.

Thanks!

**Comment by PWagner1** [comment, marked as answer]
Might be a feature request

**Reply by mbagattini** [reply]
Welp, I guess that's a no - for now 😄

**Reply by PWagner1** [reply]
@mbagattini Will be in tomorrows (31/07/2023) alpha build

**Reply by mbagattini** [reply]
Forgot to thank you for this, doing it now!
